src/commands/deploy/buildImageInAzure/buildImageInAzure.ts

```diff
@@ -0,0 +1,28 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import type { Run as AcrRun } from '@azure/arm-containerregistry';
+import { delay } from "@azure/ms-rest-js";
+import { nonNullValue } from '@microsoft/vscode-azext-utils';
+import { IBuildImageInAzureContext } from "./IBuildImageInAzureContext";
+
+const WAIT_MS = 5000;
+
+export async function buildImageInAzure(context: IBuildImageInAzureContext): Promise<AcrRun | undefined> {
+    const getRun = async () => context.client.runs.get(context.resourceGroupName, context.registryName, nonNullValue(context.run.runId));
+
+    let run = await getRun();
+    const { KnownRunStatus } = await import('@azure/arm-containerregistry');
```

Comment:
This could probably be imported at the top as `import type`

Reply:
It would be`import` not `import type` I think?

Reply:
Oh yes, I think you're right. It's an enum, not a typing.